src/main/java/org/openrewrite/docker/DockerImageVersion.java

```diff
@@ -22,11 +22,10 @@
 public class DockerImageVersion {
     String imageName;
 
-    @Nullable
     String version;
```

Comment:
Why this particular change? I don't think an empty string is quite the same as indicating clearly there was no version.
You can see this reflected further down where we now need to determine if this was an empty string before printing `:`.